install_config/cluster_metrics.adoc

```diff
@@ -1031,6 +1035,254 @@ Once `openshift_metrics_project: openshift-infra` is installed, metrics can be
 gathered from the `http://${POD_IP}:7575/metrics` endpoint.
 ====
 
+[[openshift-prometheus-kubernetes-metrics]]
+=== Kubernetes Storage Metrics via Prometheus
```

Comment:
I think we should use word {{product-title}} rather than hardcoding Kubernetes or Openshift everywhere

Reply:
We don't have an attribute for Kubernetes. The {product-title} attribute is only defined for OpenShift.

Reply:
I meant renamed "Kubernetes Storage metrics" to "Openshift storage metrics"